go/dpagg/count_test.go

```diff
@@ -339,14 +339,14 @@ func TestCountResultSetsStateCorrectly(t *testing.T) {
 }
 
 func TestCountThresholdedResult(t *testing.T) {
-	// ThresholdedResult outputs the result when it is greater than the threshold (5 using noNoise)
+	// ThresholdedResult outputs the result when it is greater than the threshold (5.00001 using noNoise)
 	c1 := getNoiselessCount()
 	for i := 0; i < 10; i++ {
 		c1.Increment()
 	}
 	got := c1.ThresholdedResult(tenten)
 	if got == nil || *got != 10 {
-		t.Errorf("ThresholdedResult(%f): when 10 addings got %v, want 10", tenten, got)
+		t.Errorf("ThresholdedResult(%f): after 2 entries got %v, want nil", tenten, got)
```

Comment:
This seems incorrect?

Reply:
Incomplete? 

343...{ 696^*= ¡ > X:Y:10.57; Axis/Directive/++ insert:;common, {(target?¿)} N^^^5= [C.E.T.A. POD] 06.16.03 3:38AM

344...Hard reboot: system override.

...if this makes any sense at all. Wait! How did I even get here?!?  😉